### DISCUSSION TITLE
how to delete bunch of user from git server

### DISCUSSION THREAD

**Discussion by vriasuhtal** [community/community#47764] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

HI Team,

I have a git server in the company, i want to remove a bunch of users that is not available in the git resources due to left the job 
how we can we remove at time this user from access list we can block or delete user but one by one manually its very hard

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @vriasuhtal ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
